pxr/usd/usdLightField/schema.usda

```diff
@@ -0,0 +1,440 @@
+             easy data footprint affordance, data consumers should prefer
+             `float` version if available.
+             """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "OrientationAttributeAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+    }
+)
+{
+    quatf[] orientations (
+        doc = "Quaternion orientation for each particle."
+    )
+    quath[] orientationsh (
+        doc = """Quaternion orientation for each particle. If the float
+                precision version is available it should be preferred."""
+    )
+}
+
+class "LightFieldScaleAttributeAPI" (
+    doc = """A ParticleField related applied schema that provides a
+             scale attribute to define the scale of the particles.
+
+             Attributes are provided in both `float` and `half` types for some
+             easy data footprint affordance, data consumers should prefer
+             `float` version if available.
+             """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "ScaleAttributeAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+    }
+)
+{
+    float3[] scales (
+        doc = """Affine scale factor applied to the base shape defined in the
+                'shape' attribute."""
+    )
+    half3[] scalesh (
+        doc = """Affine scale factor applied to the base shape defined in the
+                'shape' attribute. If the float precision version is available
+                it should be preferred."""
+    )
+}
+
+class "LightFieldOpacityAttributeAPI" (
+    doc = """A ParticleField related applied schema that provides a
+             opacity attribute to define the opacity of the particles.
+
+             The opacity here should be in the range [0, 1], and inline with the
+             traditional sense of computer graphics opacity, not the
+             transformed data sometimes seen in PLY files associated with
+             gaussian splats, where the values need to be processed with a
+             sigmoid activation function.
+
+             Attributes are provided in both `float` and `half` types for some
+             easy data footprint affordance, data consumers should prefer
+             `float` version if available.
+             """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "OpacityAttributeAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+    }
+)
+{
+    float[] opacities (
+        doc = "Opacity for each particle."
+    )
+    half[] opacitiesh (
+        doc = """Opacity for each particle. If the float precision version is
+                available it should be preferred."""
+    )
+}
+
+class "LightFieldKernelBaseAPI" (
+    doc = """Defines a base-class type applied schema that all applied schema
```

Comment:
I'd clarify that by kernel, you mean the spatial basis function supporting each particle.